MercyFork.Data/MercyForkOcktoKitExtensions.cs

```diff
@@ -0,0 +1,77 @@
+﻿using MercyFork.Data.Models;
+
+using Octokit;
+
+namespace MercyFork.Data
+{
+    public static class MercyForkOcktoKitExtensions
+    {
+        //extension method to convert RepoSearchCriteria to SearchRepositoriesRequest
+        public static SearchRepositoriesRequest ToSearchRepositoriesRequest(this RepoSearchCriteria searchCriteria)
+        {
+            return new SearchRepositoriesRequest(searchCriteria.SearchText)
+            {
+                Archived = searchCriteria.Archived,
+                Stars = searchCriteria.Stars?.ToRange(),
+                Forks = searchCriteria.Forks?.ToRange(),
+                //Followers = searchCriteria.Followers?.ToRange(),
+                SortField = searchCriteria.SortField is not null ? Enum.Parse<RepoSearchSort>(searchCriteria.SortField) : null,
+                Order = Enum.Parse<SortDirection>(searchCriteria.SortDirection),
```

Comment:
```suggestion
                Order = Enum.Parse<SortDirection>(searchCriteria.SortDirection ?? "desc"),
```
We should default to a default sort direction, passing null into the `Parse` method will throw an exception

Reply:
It will never be null. The model binding will fail if it's not provided. But it doesn't hurt to guard against it either.